spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/TcpConnection.java

```diff
@@ -125,4 +125,12 @@
 	 */
 	SSLSession getSslSession();
 
+	/**
+	 * Provides getters for {@link Socket} properties.
+	 * @return the socketInfo - may be null, for example in interceptors; interceptors
+	 * should override and delegate to the actual TcpConnection.
+	 * @since 4.3
+	 */
+	public SocketInfo getSocketInfo();
```

Comment:
All members in the interface are `public` by default.
And `static` if it isn't method :smile: 

Will fix on merge.

Minor of, course, but my IDEA warns me on the matter: like code is redundant.